spot-client/src/spot-remote/ui/components/remote-control-menu/volume-button.js

```diff
@@ -0,0 +1,52 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import { ExpandLess, ExpandMore } from 'common/icons';
+import { remoteControlService } from 'common/remote-control';
+
+const TYPE_DOWN = 'down';
+const TYPE_UP = 'up';
+
+/**
+ * Implements a volume control button.
+ */
+export default class VolumeButton extends React.Component {
+    static propTypes = {
+        type: PropTypes.oneOf(TYPE_DOWN, TYPE_UP)
+    };
+
+    /**
+     * Instantiates a new instance of the {@code Component}.
+     *
+     * @inheritdoc
+     */
+    constructor(props) {
+        super(props);
+
+        this._onClick = this._onClick.bind(this);
+    }
+
+    /**
+     * Implements {@code Component#render}.
+     *
+     * @inheritdoc
+     */
+    render() {
+        return (
+            <button
+                className = 'button volume touch-highlight'
+                onClick = { this._onClick } >
+                { this.props.type === TYPE_UP ? <ExpandLess /> : <ExpandMore /> }
+            </button>
+        );
+    }
+
+    /**
+     * Callback to handle the onClick event of the button.
+     *
+     * @returns {void}
+     */
+    _onClick() {
```

Comment:
Maybe in the future we need to be fancy and allow long-press/click-hold to go through the volume at larger increments (or do that slider).

Reply:
Yeah, since we're probably going to go for the slider thing, I wouldn't improve this screen too much now